framework/src/problems/EigenProblem.C

```diff
@@ -637,4 +639,11 @@ EigenProblem::initPetscOutput()
   _app.getOutputWarehouse().solveSetup();
 }
 
+Real
+EigenProblem::formNorm()
+{
+  mooseAssert(_bx_norm_provided,
+              "We should not get here unless a bx_norm postprocessor has been provided");
+  return getPostprocessorValueByName(getParam<PostprocessorName>("bx_norm"));
```

Comment:
Can we get the `bx_norm` in the constructor? We can assert whether the name is empty.

Reply:
I'm not quite sure what you mean. I do get `_bx_norm_provided` in the constructor. I can't get the postprocessor value itself in the constructor since the problem is constructed well before we add postprocessors

Reply:
Sorry, I mean get the name in the constructor.